src/ergw_aaa_radius.erl

```diff
@@ -737,8 +747,10 @@ radius_accounting_opts() ->
 		       ?Acct_Status_Type,
 		       ?Event_Timestamp]).
 
-send_request(Req, Session, #{server := NAS} = Opts) ->
+send_request(Req, Session, #{server := NAS, retries := Retries,
+			     timeout := Timeout} = Opts) when is_tuple(NAS)->
     Id = maps:get('NAS-Identifier', Session, <<"NAS">>),
     RadiusClientOpts = [{client_name, maps:get(client_name, Opts, Id)},
 			{server_name, maps:get(server_name, Opts, Id)}],
-    eradius_client:send_request(NAS, Req, RadiusClientOpts).
+    eradius_client:send_request(NAS, Req, RadiusClientOpts ++ 
```

Comment:
why the `++`? you could initialize the RadiusClientOpts with the values.

Reply:
Implemented.
the  async option is used here. 
https://github.com/travelping/ergw_aaa/blob/feature/add_radius_async_config/src/ergw_aaa_radius.erl#L143
Currently for the radius accounting stop msg it is always set to async by ergw.  need to configure async for accounting start msg in WUS.